isdown/assets/dashboards/isdown_overview.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
The dashboard looks good, however when your integration is published it will begin to use the `source_type_name` you've defined in the manifest.json (`isdown`) as the event source, rather than `my_apps`. The dashboard should reflect this and pull in events using` source:isdown`.